### PR TITLE
feat: booking icon offset in plan view

### PR DESCRIPTION
**Pull request by momentiris**
https://trello.com/c/BTqqUvWG/249-i-ruttkartan-visa-paket-offset-till-aktivitetsprickarna

### CODE REVIEW

**Review by izabellalarsson — COMMENTED** [review]
Is't the transport icon supposed to show on a plan?

**Comment by momentiris** [comment]
@izabellalarsson I see now that it is, yes. Will fix.

**Review by izabellalarsson — APPROVED** [review]
LGTM! 😃

**Review by lundeeell — APPROVED** [review]
Approved but route/plan is a bit confusing

File: packages/engine-ui/src/utils/mapUtils.js [hunk 1]
```diff
@@ -95,6 +86,31 @@ export const transportToFeature = (transports) => {
   }
 }
 
+export const planActivityIcon = (plan) => {
```
[the remark this answers is not in the document]
Comment (momentiris) [reply packages/engine-ui/src/utils/mapUtils.js]: Renamed it.

**Pull request merged**